lectures/10.HPC_and_parallel_compute/scripts/multiprocessing_run.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+conda activate lecture10_env
+
+# profile the serial run
+sequences_path="../data/sequences_to_analyze.txt"
+output_file_name="../results/cpg_islands_multiprocessing.csv"
```

Comment:
would these actually be islands in the example, or are they simply instances of the specific sequence?

Reply:
Instances of a 5mC in a sequence